tox.ini

```diff
@@ -20,8 +21,10 @@ deps=
     django40: django~=4.0.0
     django41: django~=4.1.0
     django42: django~=4.2.0
+    django50: django~=5.0b1
     cms311: https://github.com/yakky/django-cms/archive/release/3.11.x.zip
     cms311: djangocms-text-ckeditor>=5.0,<6.0
+    py3.12: setuptools
```

Comment:
we can use https://peps.python.org/pep-0508/#complete-grammar to declare setuptools for python 3.12 and up

See https://stackoverflow.com/a/73382385

Update classifiers too

Reply:
@yakky Cool, didn't know about that. Done!